### PR TITLE
Added 1.3.4 release

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Added 1.3.4 release

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _data/versions.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-  "current": "1.3",
+  "current": "2.1",
```
Comment (AMoo-Miki) [inline _data/versions.json:2]: This is the 1.3 branch. `current` should remain `1.3`.

File: _opensearch/install/plugins.md [hunk 2]
```diff
@@ -63,27 +63,27 @@ bin/opensearch-plugin list
...
+opensearch-cross-cluster-replication 2.1.0.0
+opensearch-index-management          2.1.0.0
+opensearch-job-scheduler             2.1.0.0
+opensearch-knn                       2.1.0.0
+opensearch-ml                        2.1.0.0
+opensearch-observability             2.1.0.0
+opensearch-performance-analyzer      2.1.0.0
+opensearch-reports-scheduler         2.1.0.0
+opensearch-security                  2.1.0.0
+opensearch-sql                       2.1.0.0
```
Comment (AMoo-Miki) [inline _opensearch/install/plugins.md:80]: This is the 1.3 branch; should it even acknowledge that a v2 exists?

**Review by AMoo-Miki — CHANGES_REQUESTED** [review]

File: _data/versions.json (hunk 1 above)
Comment (kolchfa-aws) [reply _data/versions.json:2]: Done

File: _opensearch/install/plugins.md (hunk 2 above)
Comment (kolchfa-aws) [reply _opensearch/install/plugins.md:80]: Removed v2

**Pull request closed without merging**